config/overlays/make-deploy/kustomization.yaml

```diff
@@ -3,3 +3,9 @@ kind: Kustomization
 namespace: opendatahub
 resources:
 - ../../base
```

Comment:
Actually, instead you can just add the argo resources here instead `- ../../argo` and the `make deploy` will essentially become `make deployODH` which I believe is the point of this PR.